src/main/java/uk/gov/hmcts/reform/hmc/api/model/ccd/HearingChannelsEnum.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.hmcts.reform.hmc.api.model.ccd;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import lombok.RequiredArgsConstructor;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+@RequiredArgsConstructor
+@JsonSerialize(using = CustomEnumSerializer.class)
+public enum HearingChannelsEnum {
+    @JsonProperty("INTER")
```

Comment:
can we discuss on this as i am not fully sure whether we need all the values

Reply:
sure @yogendraU not needed all of them will check with you and remove nonRequired values please